src/stellar/index.js

```diff
@@ -22,6 +22,8 @@ module.exports = async function (models) {
     callBuilder.cursor(latestTx.cursor)
   }
 
+  // What happens here if latestTx is null for any reason?
```

Comment:
Do not add questions in the source :-)

If it's null we start at the first incoming entry of the account, which is a good idea because null would happen if we haven't collect anything yet.

Reply:
No problem. What's the best way for me to add questions like this? Something through github or just contact through Slack?